packages/accounts-password/password_tests_config.js

```diff
@@ -0,0 +1,4 @@
+// Set useLowerCaseEmails's value to false to prevent lowercase email enforcement in the system.
```

Comment:
Wrong comment, the settings is set to `true`. Also aren't test missing in this file?

Reply:
@StorytellerCZ  I guess the comment is not explanatory. I will fix this. I wanted to say that if you explicitly set the value of uselowercaseEmails to false then the system will not enforce lowercase emails or else by default it will implement lower case email enforcement.

This file is only a config setup the tests are being done in password test and email test file. I have not written any new tests as the present tests were covering all use cases that i could think of. So i mostly modified the exisiting tests to accomodate for lowercase enforcement.

Some tests where checking for case insensitive email which is not applicable if the system enforce lower case email, so I shifted those tests inside an if condition which check the value of uselowercaseEmails and if the value is explicitly set to false, implement these test. I tested it in my local system by toggling the useLowerCaseEmails value to false  and true and it worked.

Reply:
Another approach would be to run the tests with the option set to true and then again when set to false (minus the case sensitive tests you have mentioned).